mkt/reviewers/forms.py

```diff
@@ -200,7 +200,7 @@ def __init__(self, *args, **kwargs):
     def clean_status(self):
         status = self.cleaned_data['status']
         if status == 'any':
-            return 'any'
+            return None
```

Comment:
We used to explicitly check for not `'any'` to apply the filter (https://github.com/mozilla/zamboni/pull/2906/files#diff-9a26e6efe2570209f70b06294f23ca39L1007) but now we check for not `None` (https://github.com/mozilla/zamboni/pull/2906/files#diff-26e55b38d2cbc410af290d209506f3d7R120)